test/evergreen.yml

```diff
@@ -608,7 +608,7 @@ functions:
         source venv/bin/activate
         pip3 install lxml==4.8.0 Pygments==2.11.2 Jinja2==3.0.3 gcovr==5.0
         mkdir -p ../coverage_report
-        GCOV=/opt/mongodbtoolchain/v4/bin/gcov gcovr -r .. -f ../src -e '.*/bt_(debug|dump|misc|salvage|vrfy).*' -e '.*/(log|progress|verify_build|strerror|env_msg|err_file|cur_config|os_abort)\..*' -e '.*_stat\..*' -e 'bench' -e 'examples' -e 'test' -e 'ext' -e 'dist' -e 'tools' -j 4 --html-self-contained --html-details ../coverage_report/2_coverage_report.html --json-summary-pretty --json-summary ../coverage_report/1_coverage_report_summary.json
+        GCOV=/opt/mongodbtoolchain/v4/bin/gcov gcovr -r .. -f ../src -e '.*/bt_(debug|dump|misc|salvage|vrfy).*' -e '.*/(log|progress|verify_build|strerror|env_msg|err_file|cur_config|os_abort)\..*' -e '.*_stat\..*' -e 'bench' -e 'examples' -e 'test' -e 'ext' -e 'dist' -e 'tools' -j 4 --html-self-contained --html-details ../coverage_report/2_coverage_report.html --json-summary-pretty --json-summary ../coverage_report/1_coverage_report_summary.json --json ../coverage_report/full_coverage_report.json
```

Comment:
@jeremythorp Is there any patch where I could see the content of the full_coverage_report.json file and how it is different than other JSON summary files?

Reply:
Hi @raviprakashgiri29 , the key difference for the 'full' report is that it provides line-by-line branch coverage for every file, and that's a lot more data than the summary. As an example, see
https://spruce.mongodb.com/task/wiredtiger_code_statistics_coverage_report_catch2_patch_15cdc93410da79cb6c23a2b05028abe7e8ee7642_650788f9562343f6199266e9_23_09_17_23_17_15/files?execution=0 and 
https://build_external.s3.amazonaws.com/wiredtiger/code-statistics/15cdc93410da79cb6c23a2b05028abe7e8ee7642/coverage_report_wiredtiger_code_statistics_patch_15cdc93410da79cb6c23a2b05028abe7e8ee7642_650788f9562343f6199266e9_23_09_17_23_17_15-0/full_coverage_report.json